lighthouse-core/config/v2/default.json

```diff
@@ -0,0 +1,308 @@
+{
+  "version": 2,
+
+  "gatherers": {
+    "url": {},
+    "https": {},
+    "viewport": {},
+    "theme-color": {},
+    "manifest": {},
+    "accessibility": {},
+    "image-usage": {},
+    "content-width": {},
+    "service-worker": {},
+    "offline": {},
+    "http-redirect": {},
+    "html-without-javascript": {},
+    "chrome-console-messages": {},
+    "styles": {},
+    "css-usage": {},
+    "all-event-listeners": {
+      "path": "dobetterweb/all-event-listeners"
```

Comment:
Not sure what you're trying to say here. IMO, this is actually a step in the direction of decoupling our arbitrary repo folder structure from magic happening in the config. What leads you to believe our repo organization process is now inherent to the gatherer now?